github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -782,6 +782,40 @@ periodics:
           memory: 8Gi
       securityContext:
         privileged: true
+- annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+  cluster: s390x-prow-virtual
+  cron: 40 5 * * *
+  decorate: true
+  decoration_config:
+    grace_period: 5m0s
+    timeout: 2h0m0s
+  extra_refs:
+  - base_ref: main
+    org: kubevirt
+    repo: kubevirt
+  labels:
+    preset-bazel-unnested: "false"
```

Comment:
This should be set to true as we don't have a builder image available for s390x yet. 

I was able to run the unit tests - with unnested set to true.